### ISSUE TITLE
Problem with oversize map generation

### ISSUE THREAD

**Issue by SuperStucco** [MegaMek/mekhq#3178]
In the attached 49.7 campaign, a 'Low Atmo - Destroy DropShip' scenario has been generated with a problematic map size: 50 x 700.

Even though the friendly forces are a full star of 10 aerospace fighters, this is excessive.  And since the 700-hex length is in the vertical the minimap is useless as most of it extends off the bottom of the screen.  Additionally the enemy DropShip is moving west-to-east across 50 hexes; even though its a spheroid type it's virtually impossible to close and do enough damage to stop it in that distance.

[Green Garrison Cluster30510915.cpnx.gz](https://github.com/MegaMek/mekhq/files/8312069/Green.Garrison.Cluster30510915.cpnx.gz)

Some custom infantry units are present, same as in this post: https://github.com/MegaMek/mekhq/issues/3160#issue-1160710782

**Comment by IllianiCBT** [comment]
I don't believe this will be an issue with the new scenarios, so I'm going to go ahead and close it. However, if the issue crops up again feel free to reopen the case. :)
